packages/extension-chakra-storefront/src/components/search/index.jsx

```diff
@@ -181,16 +189,38 @@ const Search = (props) => {
         shouldOpenPopover()
     }
 
+    const onClearSearch = () => {
+        if (searchInputRef.current) {
+            searchInputRef.current.value = ''
```

Comment:
Directly mutating the DOM seems anti-react...

Reply:
Here, we are using input in a uncontrolled way where we use ref instead of useState. It is still a React pattern.